### ISSUE TITLE
Python compatible?

### ISSUE THREAD

**Issue by lab313ru** [rxi/dyad#17]
Hi. Is it possible to use Python sockets to contact with C side that uses dyad?

**Comment by rxi** [comment]
Yes — If you build and run the [echo server example](https://github.com/rxi/dyad/blob/master/example/echoserv.c), the following python script could be used on the same machine to connect to it:

```python
import socket

s = socket.socket()
s.connect(("localhost", 8000))

s.send("Hello world\n")

print(s.recv(100))
```

**Comment by cmarshall108** [comment]
You can write methods in c that are wrapped using the CPython API, therefore you can call those methods straight in python.
